Task7_Razor_AWS_SQS_SNS/Pages/FileUpload.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@page
+@model Task7_Razor_AWS_SQS_SNS.Pages.FileUploadModel
+@{
+}
+<div class="text-center">
+    <h1 class="display-4">File Upload</h1>
+</div>
+
+<div class="container">
+    <div class="mb-3 mt-3">
+        <form method="post" enctype="multipart/form-data">
+            <div class="mb-3">
+                <label class="form-label">Upload File</label>
+                <input type="file" name="excelFile" class="form-control" />
+            </div>
+            <div class="mb-3">
+                <button type="submit" class="btn btn-primary">Upload</button>
+            </div>
+        </form>
+    </div>
+    <div id="messageContainer"></div>
+</div>
+
+<script>
+    document.querySelector('form').addEventListener('submit', function (e) {
+        const fileInput = document.querySelector('input[type="file"]');
+        const fileName = fileInput.value;
+        if (!fileName) {
+            e.preventDefault();
+            alert('Please select a file to upload.');
```

Comment:
Keep your messages in a separate file. So change in one file would be reflected to everywhere else.

Reply:
Noted.